com.airtame.Client.appdata.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.airtame.Client</id>
+  <name>Airtame</name>
+  <summary>Airtame is a wireless streaming solution</summary>
+  <developer_name>Airtame</developer_name>
+  <description>
+  <p>Airtame is a small wireless HDMI device that plugs into the HDMI port of any screen or projector. Unlike other wireless HDMI devices, nothing is plugged into your computer or smartphone. Simply download our app and stream your content to the screen from any major computer platform, tablet, or smartphone.</p>
+</description>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <url type="help">https://help.airtame.com/</url>
+  <url type="homepage">https://airtame.com/</url>
```

Comment:
Do they provide any bugtracker or help url? it could be useful for users

Reply:
> Do they provide any bugtracker or help url? it could be useful for users

I have already added `<url type="help">https://help.airtame.com/</url>`. They don't have a bugtracker as far as I know.

Reply:
Great. thanks!